launch/state_machines/challenge_final.launch

```diff
@@ -13,18 +13,25 @@
 			<arg name="faces_db" value="~/MEGA/data/$(optenv ROBOT_ENV rgo2018)/models/openface_ros/faces.pkl" />
 		</include>
 
-		<!-- Joke -->
-		<include file="$(find robot_launch_files)/launch/essentials/joke.launch">
-			<arg name="machine" value="amigo2" />
-		</include>
+		<!-- Launch the websocket server (communication via roslib) -->
+		<node name="rosbridge_websocket_amigo2" pkg="rosbridge_server" type="rosbridge_websocket" output="log">
+			<param name="authenticate" value="false" />
+			<param name="port" value="9090"/>
+			<param name="address" value=""/>
+			<remap from="/amigo/hmi/string" to="/amigo/message_to_ros" />
+		</node>
 
-		<!-- telegram face recognizer -->
-		<node name="telegram_face_recognizer" pkg="challenge_final" type="telegram_face_recognizer.py" output="log" respawn="true" />
+		<!-- Start the simple webserver to handle static files -->
+		<node pkg="tue_mobile_ui" type="webserver.py" name="webserver_amigo2" output="log">
```

Comment:
Same holds for this

Reply:
Depent on https://github.com/tue-robotics/tue-env/issues/200 ?

Reply:
Now that tue-robotics/tue-env#200 is solved. What should be done now?

Reply:
Then one webserver should suffice

Reply:
@LoyVanBeek I think webserver also Neede to be removed.